.github/workflows/validate-json.yaml

```
@@ -0,0 +1,32 @@
+name: Validate emojibase.json
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    concurrency:
+      # When running on main, use the sha to allow all runs of this workflow to run concurrently.
+      # Otherwise only allow a single run of this workflow on each branch, automatically cancelling older runs.
+      group: ${{ github.ref == 'refs/heads/main' && format('validate-json-main-{0}', github.sha) || format('validate-json-pr-{0}', github.ref) }}
+      cancel-in-progress: true
```

```suggestion
      # When running on main, use the sha to allow all runs of this workflow to run concurrently.
      # Otherwise only allow a single run of this workflow on each branch, automatically cancelling older runs.
      group: ${{ github.workflow }}
      cancel-in-progress: ${{ github.ref_name != 'main' }}
```

This seems simpler yet and has the benefit of not running an unbound number of `main` branch builds in parallel causing runner starvation for the whole org.